pkg/partition/partition_test.go

```diff
@@ -15,111 +15,111 @@ func TestPartition(t *testing.T) {
 	v0 := vector.New(types.Type{Oid: types.T(types.T_int8)})
 	v0.Data = encoding.EncodeInt8Slice([]int8{3, 4, 5, 6, 7, 8})
 	v0.Col = encoding.DecodeInt8Slice(v0.Data)
-	v0.Nsp = &nulls.Nulls{Np: roaring.New()}
+	v0.Nsp = &nulls.Nulls{}
 	partitions := make([]int64, 2)
 	Partition([]int64{1, 3, 5}, []bool{false, false, false}, partitions, v0)
 	require.Equal(t, []int64{0, 1}, partitions)
-	v0.Nsp.Np.Add(1)
+	nulls.Add(v0.Nsp, 1)
```

Comment:
Why you like nulls.Add(v0.Nsp, 1)               
Can vector has a method v0.SetNull(i int)    and we do v0.SetNull(1)?